CMS-Enterprise/mint-app
# [MINT-3249] Edit Category Titles From Action Menu

# MINT-3249

## Description

* Add `EditCategoryTitleForm` and added to `MTOModal`
* Changed `ActionMenu` to pass props and open MTOModal with EditCategory stuff
* Created `errorMessage` and `errorMessageInModal` in `MessageContext`. see comment below: https://github.com/CMS-Enterprise/mint-app/pull/1576#discussion_r1890514007
* Plugged in `errorMessage` and `errorMessageInModal` throughout the MTOModal forms




## How to test this change
1. Navigate to MTO Matrix
2. Click `...` in any category/subcategory and `Edit sub/category title`
3. Ensure that the modal popping up matches figma
4. Verify functionality
5. Try to break it


## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
